webviz_config/templates/webviz_template.py.jinja2

```diff
@@ -117,12 +119,8 @@ else:
                     href="/",
                     {%- else -%}
                     "{{page.title}}",
-                    # We will create a webviz-core-components
-                    # component instead of styling dcc.Link's,
-                    # then we can more easily change className to
-                    # selectedButton for current page.
                     className="styledButton",
-                    id="{{page.id}}",
+                    id={"id":"{{page.id}}","type":"page_child"},
```

Comment:
```suggestion
                    id={"id": "{{page.id}}", "type": "page_child"},
```
In order to reduce risk of name conflicts with potential plugin pattern-matching callbacks, should we increase the `type` name detail level? `webviz_config_main_sidebar_link`?